W 22008605 vibes new features fa (#620)
mulesoft/docs-code-builder: modules/ROOT/pages/vibes-workflows-commands.adoc

Rules and Workflows allow developers to customize how Mulesoft Vibes operates within their project.

Rules define natural-language constraints that MuleSoft Vibes follows when generating content.

> You can use rules to enforce organizational conventions, ensure consistent error handling, and maintain coding standards across your workspace.

Reviewer: ```suggestion
Use rules to enforce organizational conventions, ensure consistent error handling, and maintain coding standards across your workspace.
```

Workflows that you define also appear as commands under *Workflow Commands*.  
You can trigger them the same way by typing `/` followed by the workflow name.

== Enforce Organizational Conventions with Rulesets

> Rulesets allow you to define and apply organizational conventions across your development workflows using natural language.

Reviewer: ```suggestion
Use rulesets to define and apply organizational conventions across your development workflows using natural language.
```

> Rulesets allow you to define and apply organizational conventions across your development workflows using natural language.

Reviewer 2: ```suggestion
Define and apply organizational conventions with rulesets across your development workflows using natural language.
```
